wemake_python_styleguide/visitors/ast/loops.py

```diff
@@ -208,22 +178,21 @@ def _check_multiline_loop(self, node: _AnyLoop) -> None:
 
     def _has_break_raise_or_return(self, node: _AnyLoop) -> bool:
```

Comment:
This method can still be moved into `logic/`

Reply:
Sorry, that function should not have been in this commit in that location. It has been moved to logic/.